airflow-code-editor/airflow_code_editor/app_builder_view.py

```diff
@@ -98,68 +102,9 @@ def _render(self, template, *args, **kargs):
                 **kargs
             )
 
-
 except (ImportError, ModuleNotFoundError):
-    from airflow_code_editor.auth import has_access
```

Comment:
This was support for older airflow versions which we don't intent to, so removing for cleanliness.